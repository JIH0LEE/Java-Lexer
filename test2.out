<VTYPE,int>
<ID,main>
<LPAREN>
<RPAREN>
<LBRACE>
<VTYPE,char>
<ID,if123>
<ASSIGN>
<CHARACTER,'1'>
<SEMICOLON>
<VTYPE,int>
<INTEGER,0>
<ID,a>
<ASSIGN>
<ID,a>
<OP,+>
<INTEGER,-1>
<SEMICOLON>
<RETURN>
<OP,->
<INTEGER,0>
<SEMICOLON>
<RBRACE>
