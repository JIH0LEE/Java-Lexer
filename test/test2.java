String getStatus(int age){
    String status;
    if(true){
        status="old";
    } 
    else{
        status="young";
    }
    return status;
}
