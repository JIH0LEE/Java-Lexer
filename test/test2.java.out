<1,VTYPE,String>
<1,ID,getStatus>
<1,LPAREN,(>
<1,VTYPE,int>
<1,ID,age>
<1,RPAREN,)>
<1,LBRACE,{>
<2,VTYPE,String>
<2,ID,status>
<2,SEMI,;>
<3,IF,if>
<3,LPAREN,(>
<3,BOOLSTR,true>
<3,RPAREN,)>
<3,LBRACE,{>
<4,ID,status>
<4,ASSIGN,=>
<4,LITERAL,"old">
<4,SEMI,;>
<5,RBRACE,}>
<6,ELSE,else>
<6,LBRACE,{>
<7,ID,status>
<7,ASSIGN,=>
<7,LITERAL,"young">
<7,SEMI,;>
<8,RBRACE,}>
<9,RETURN,return>
<9,ID,status>
<9,SEMI,;>
<10,RBRACE,}>
